docs/compatibility.md

```diff
@@ -277,5 +277,73 @@ However, Spark may produce different results for a compiled udf and the non-comp
 
 When translating UDFs to Catalyst expressions, the supported UDF functions are limited:
 
-| Operand type                                                | Operation |
-| ------------------------------------------------------------------- | ------------------|
+| Operand type             | Operation                                                |
```

Comment:
I'd prefer it if these were multiple tables, one per "Operand type", with a description column. The description column could be redundant in most cases, but it may allow you to have a place to put extra limitations or issues for each operation supported. 

### Arithmetic Unary
| Operation | Description   |
| ------------- | ----------------- |
| +x             | Unary plus    |
| -x              | Unary minus |

### Arithmetic Binary
...

Reply:
I'm wondering if we should put those tables into another md doc. As they are related to only the udf plugin and the table can be very long.

Reply:
@wjxiz1992 Having another page for the UDF compiler may be what we end up doing, but I wouldn't worry about doing that yet. I would just continue where you are and try to follow the formatting in that page (https://nvidia.github.io/spark-rapids/docs/compatibility.html). This is very much relevant to the compatibility section.